src/components/live-viewers/visible-icon.tsx

```diff
@@ -0,0 +1,10 @@
+import {h} from 'preact';
+export const VisibleIcon = () => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
+      <path fill-rule="evenodd" clip-rule="evenodd"
+            d="M8.00001 3.33337C11.0303 3.33337 13.6182 5.07337 14.6667 8.00004C13.6182 10.9267 11.0303 12.6667 8.00001 12.6667C4.96971 12.6667 2.38183 10.9267 1.33334 8.00004C2.38183 5.07337 4.96971 3.33337 8.00001 3.33337ZM8.00001 4.66671C6.16297 4.66671 4.66668 6.163 4.66668 8.00004C4.66668 9.83708 6.16297 11.3334 8.00001 11.3334C9.83705 11.3334 11.3333 9.83708 11.3333 8.00004C11.3333 6.163 9.83705 4.66671 8.00001 4.66671ZM10 8.00004C10 6.89782 9.10223 6.00004 8.00001 6.00004C6.89779 6.00004 6.00001 6.89782 6.00001 8.00004C6.00001 9.10226 6.89779 10 8.00001 10C9.10223 10 10 9.10226 10 8.00004Z"
+            fill="white"/>
```

Comment:
should the fill always be white ? does it need to be customizable ?

Reply:
no.